Update VD POC (#7215)
wazuh/wazuh-documentation: source/proof-of-concept-guide/poc-vulnerability-detection.rst

      $ sudo systemctl restart wazuh-manager

Test the configuration
----------------------

> You don’t have to perform any action. It performs vulnerability detection scans periodically for applications and operating systems on each monitored endpoint. In this example we use the installed packages on the Ubuntu endpoint.

Reviewer: ```suggestion
You don’t have to perform any action. Wazuh detects the vulnerable packages installed in the Ubuntu endpoint automatically. The time it takes to detect vulnerabilities depends on the Syscollector update interval configured in the ``ossec.conf`` file.
```

Reviewer: Consider adding a link to the interval syscolletor option referred.

Author: Done!

> 

Reviewer: ```suggestion
```

Author: Done!

> .. note::

Reviewer: ```suggestion
```

Author: Done!

>    The time it takes to detect vulnerabilities depends on the syscollector update interval configured in the ``ossec.conf`` file.

Reviewer: ```suggestion
```

Author: Done!

Visualize the alerts
--------------------

> You can visualize the alert data in the Wazuh dashboard. To do this, go to the Vulnerability Detection module, select Inventory, and see all vulnerable packages. Use the search bar to filter the results.

Reviewer: ```suggestion
You can visualize detected vulnerabilities data in the Wazuh dashboard. Go to **Vulnerability Detection** and select **Inventory** to see a list of all vulnerable packages detected. Add ``firefox`` in the search bar to filter vulnerable Firefox packages.
```

Author: Done!

.. thumbnail:: /images/poc/vulnerabilities-inventory.png

>    :title: Detected vulnerabilities on Ubuntu, firefox example

Reviewer: ```suggestion
   :title: Detected vulnerabilities on Ubuntu. Vulnerable Firefox packages example
```

Author: Done!

      $ sudo systemctl restart wazuh-manager

Test the configuration
----------------------

You don’t have to perform any action. Wazuh detects the vulnerable packages installed in the Ubuntu endpoint automatically. The time it takes to detect vulnerabilities depends on the Syscollector :doc:`interval </user-manual/reference/ossec-conf/wodle-syscollector>` configured in the ``ossec.conf`` file.

#. Install ``vim`` in the Debian endpoint with ``sudo apt install vim -y``.

> #. Wait until the syscollector runs a new scan.

Reviewer: ```suggestion
```

> #. Delete the package with ``sudo apt purge vim -y``.

Reviewer: ```suggestion
#. Delete the package with ``sudo apt purge vim -y`` to fix the vulnerability.
```
